<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оптимизация границ</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>521</width>
     <height>306</height>
    </rect>
   </property>
   <property name="title">
    <string>Таблицы </string>
   </property>
   <widget class="QTableWidget" name="tableView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>511</width>
      <height>261</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Таблица</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>xmin</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ymin</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>xmax</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ymax</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QCheckBox" name="ch_box_bounds_type">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>285</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Границы по геометрии</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_close">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>315</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Первая выбраная таблица становится базовой </string>
   </property>
   <property name="title">
    <string>Основная таблица</string>
   </property>
   <widget class="QLabel" name="lb_name_tab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>236</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Таблица не выбрана</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_base_cs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>286</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Проекция </string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_MergeBound">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>315</y>
     <width>241</width>
     <height>126</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Общиq bound выбранных таблиц</string>
   </property>
   <property name="title">
    <string>Общая граница (bound)</string>
   </property>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>35</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>60</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_xmin">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>35</y>
      <width>96</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_ymin">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>60</y>
      <width>96</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_ymax">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>96</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_xmax">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>35</y>
      <width>96</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_recalc">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Пересчитать</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_run">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_out">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>445</y>
     <width>521</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Оптимизировать и сохранить копии</string>
   </property>
   <widget class="QSlider" name="hz_slider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_edit_proc">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_out_path">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>65</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_change_path">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>60</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Изменить путь файла(директории)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/saveas.png</normaloff>icon/saveas.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
